src/logic/calculate.js

```diff
@@ -0,0 +1,100 @@
+import operate from './operate';
+
+const calculate = ({ total, next, operation }, buttonName) => {
+  const operations = ['%', '÷', '+', '-', 'X'];
+  const nums = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  if (buttonName === 'AC') {
```

Comment:
**[OPTIONAL]**
It relatively faster to use `if/else if` statement in this case. In this case, all the condition would run, It is said that this effect insignificant when the code is small, but I just wanted to draw to your attention.

Reply:
can't use the else if statement cause it prompts an error on the linters:
![image](https://user-images.githubusercontent.com/34653764/102550163-a4a1dd00-40bd-11eb-9866-9dae9a7e82e0.png)